crates/rmf_site_editor/src/site/assets.rs

```diff
@@ -32,13 +32,29 @@ pub(crate) fn add_site_icons(app: &mut App) {
         embedded_asset!(app, "src/", "icons/parking.png");
         embedded_asset!(app, "src/", "icons/stopwatch.png");
     }
-
+    
     #[cfg(all(target_family = "windows", not(target_env = "gnu")))]
     {
         embedded_asset!(app, "src\\", "icons\\battery.png");
         embedded_asset!(app, "src\\", "icons\\parking.png");
         embedded_asset!(app, "src\\", "icons\\stopwatch.png");
     }
+
+    #[cfg(any(not(target_family = "windows"), target_env = "gnu"))]
+    {
+        embedded_asset!(app, "src/", "icons/baseBillboard.png");
+        embedded_asset!(app, "src/", "icons/chargingBillboard.png");
+        embedded_asset!(app, "src/", "icons/parkingBillboard.png");
+        embedded_asset!(app, "src/", "icons/holdingBillboard.png");
+    }
+
+    #[cfg(all(target_family = "windows", not(target_env = "gnu")))]
+    {
+        embedded_asset!(app, "src\\", "icons\\baseBillboard.png");
+        embedded_asset!(app, "src\\", "icons\\chargingBillboard.png");
+        embedded_asset!(app, "src\\", "icons\\parkingBillboard.png");
+        embedded_asset!(app, "src\\", "icons\\holdingBillboard.png");
+    }
```

Comment:
Note this shouldn't be needed anymore (check https://github.com/open-rmf/rmf_site/pull/357)